config/default/config_ignore.settings.yml

```diff
@@ -1,8 +1,10 @@
 _core:
   default_config_hash: UVH1aJ4b44UM-VdPVN7hNNuuVqfReJxwfVeDQH1Hvsk
 ignored_config_entities:
-  - 'system.site:page.front'
+  - fathom_analytics.settings
+  - system.site
```

Comment:
Why not `system.site:page.front`?

Reply:
Changed to system.site:page to catch other pages